Changelog.md

```diff
@@ -0,0 +1,60 @@
+# Toolkit Changelog
+
+## Process for updating this change log (Proposal)
+This change log should be updated as part of a PR if the work is worth noting (most of them should be). If unsure, always add an entery here for any PR targeted for the next release. It's easier to remove than add an entry at final review time for the next release.
+
+## Next Release (Date TBD)
+
+#### New experimental features
+
+#### Features stabilized
```

Comment:
nit: I'd prefer "Stabilized features" in keeping with the adjective-noun style that the rest of the headings use

Reply:
👍